### DISCUSSION TITLE
Output metrics for later plotting and analysis

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [ClementTsang/bottom#1384] [General]
Is there a way to have `bottom` export its metrics in a file/log for later checking?

My reason for this is that i would like to have an idea of how my battery is being drained , how fast , which process is running at that point and what not.

I don't know of a better tool than `bottom` for monitoring system , and thank you for that, and i thought if it could output somewhere and i could collect any data i want from the generated dataset!

example: 
```console
bottom --monitor "battery,cpu,processes" > monitor-output.'extension'
```

**Comment by ClementTsang** [comment]
Currently no way to do so, though that might change in the future.

**Reply by LamprosPitsillos** [reply]
Thanks for the consideration,Would you like me to open a feature request issue?

**Reply by ClementTsang** [reply]
Sure, feel free to.

**Comment by Bit0r** [comment]
@LamprosPitsillos  Meta has open-sourced a time-traveling monitor, but there is no deb package and it is very difficult to compile successfully.

[below](https://github.com/facebookincubator/below)
